ddtrace/contrib/django/patch.py

```diff
@@ -435,7 +435,7 @@ def traced_as_view(django, pin, func, instance, args, kwargs):
     except Exception:
         log.debug("Failed to instrument Django view %r", instance, exc_info=True)
     view = func(*args, **kwargs)
-    return wrapt.FunctionWrapper(view, traced_func(django, "django.view", resource=func_name(view)))
+    return wrapt.FunctionWrapper(view, traced_func(django, "django.view", resource=func_name(instance)))
```

Comment:
view.__class__.__name__ returns "view" in django 4.0 instead of the name of the sub class. According to [this comment](https://github.com/django/django/blob/main/django/views/generic/base.py#L74), django 4.0 supports accessing the name of a View using the`view_class` attribute. Instead of this approach this change access the class name using the instance object of a view. This approach is backwards compatible and can be used with the existing `func_name` util